bin/obj2gltf.js

```diff
@@ -49,6 +49,12 @@ var argv = yargs
             describe: 'Quantize positions, compress texture coordinates, and oct-encode normals.',
             type: 'boolean'
         },
+        'optimize': {
+            alias: 'p',
+            default: false,
+            describe: 'Turn the optimization pipeline stages on/off to preserve the original glTF hierarchy.',
+            type: 'boolean'
+        },
```

Comment:
Use the description in the README here, as I think that one is clearer.

`Use the optimization stages in the glTF pipeline`.

Also to avoid confusion `-p` should be a different letter, since that flag means the opposite effect in the gltf-pipeline. `-z` could be fine.